components/Ecosystem/EcosystemCardTestResults.vue

```diff
@@ -1,12 +1,7 @@
 <template>
   <UiDataTable
     class="ecosystem-card-test-results"
-    :columns="[
-      'Status',
-      'Test Type',
-      `${tableData[0][2].packageName} Version`,
-      'Test Logs',
-    ]"
+    :columns="['Status', 'Test Type', `Qiskit Version`, 'Test Logs']"
```

Comment:
We can use `'` here:

```suggestion
    :columns="['Status', 'Test Type', 'Qiskit Version', 'Test Logs']"
```